cogip/tools/planner/__main__.py

```diff
@@ -12,7 +12,7 @@
 
 
 def changes_callback(changes):
-    logger.info("Changes detected:", changes)
+    logger.info(f"Changes detected: {changes}")
```

Comment:
In commit:
```
tools: planner: re-enable auto-reload

After new tests, it seems to work fine.
```

Can you update your commit body to explain what and why it works fine.

Reply:
I don't know, it is not linked to this PR.
Improvements in async loops and subprocess exit have been done in previous PR, as a result, auto-reload is now
now working correctly enough to be activated again.